src/Modules/addAndremove.js

```diff
@@ -0,0 +1,74 @@
+const demo = () => {
+  let value = 0;
+  const array = [];
+  const btnArray = [];
+  let i = 0;
+  let id = 10000;
+  const inputHolder = document.querySelector('.input-holder');
+  inputHolder.addEventListener('keyup', (event) => {
+    const li = document.createElement('li');
+    li.className = 'to-do-list-item';
+    const ulElement = document.querySelector('.li-item-container');
+    if (event.keyCode === 13 && inputHolder.value.trim().length > 0) {
+      const obj = {
+        description: inputHolder.value,
+        complete: false,
+        index: (value += 1),
+      };
+      array.push(obj);
+      const div = document.createElement('div');
+      const listDescription = document.createElement('label');
+      listDescription.className = 'list-description';
+      div.classList.add('checkbox-and-description');
+      const checkbox = document.createElement('button');
+      checkbox.type = 'checkbox';
+      checkbox.className = 'checkbox';
+      div.appendChild(checkbox);
+      const idValue = `cb${id += 1}`;
+      li.id = idValue;
+
+      listDescription.innerHTML = array[i++].description;
+      listDescription.classList.add(false);
+      localStorage.setItem(idValue, inputHolder.value);
+      div.appendChild(listDescription);
+      listDescription.addEventListener('click', () => {
+        listDescription.contentEditable = true;
+        listDescription.focus();
+      });
```

Comment:
Great work with your task `update` logic. 🔥 However, there is a tiny suggestion I would want to make to make it perfect. 💯 

Currently the only means to confirm a task update is to click outside the focus area of the task. 🎯 

- [ ] Please make sure that `ENTER` `key` or `button` can also save an update. 💾